test/workbox-cache-expiration/node/test-Plugin.mjs

```diff
@@ -68,6 +69,30 @@ describe(`[workbox-cache-expiration] Plugin`, function() {
       });
       expect(plugin._config.maxEntries).to.equal(10);
     });
+
+    it(`should register a quota error callback when purgeOnQuotaError is true`, async function() {
+      const plugin = new Plugin({
+        maxEntries: 10,
+        purgeOnQuotaError: true,
+      });
+      plugin.deleteCacheAndMetadata = sandbox.stub();
+
+      await executeCallbacks();
```

Comment:
Same comment here as above w.r.t the name.

Reply:
Not sure what you mean—the name of what?

Reply:
This is definitely a nit, but I mention it because it confused me a bit when reading the code for the first time. 

Basically my point is that the name `executeCallbacks()` here has no context, so as a reader you have to go searching for where it's defined/imported to figure out what it's doing.

If it were called something like `executeQuotaErrorCallbacks()` or something like that, then it'd be more clear. Alternatively, if you imported the entire namespace as `quotaErrors`, then you'd have more context because it'd be `quotaErrors.executeCallbacks()`.

Anyway, like I said this is a nit. If you think it's fine, I'm OK with it.

Reply:
Note, this comment primarily applies to [line 90 in `cacheWrapper`](https://github.com/GoogleChrome/workbox/pull/1505/files/b09566d00f13e1e709423d84af609dabd5d89c3d#diff-0e280af723edc4482219279065921834R90), not necessarily here.

Reply:
Renamed to `executeQuotaErrorCallbacks()`.